src/server/multi_test.cc

```diff
@@ -657,22 +657,22 @@ TEST_F(MultiTest, ScriptConfig) {
   }
 }
 
-TEST_F(MultiTest, ScriptPragmas) {
+TEST_F(MultiTest, ScriptFlags) {
   const char* s1 = R"(
-  -- pragma: allow-undeclared-keys
+  #!lua flags=allow-undeclared-keys
   return redis.call('GET', 'random-key');
 )";
 
-  // Check eval finds script pragmas.
+  // Check eval finds script flags.
   Run({"set", "random-key", "works"});
   EXPECT_EQ(Run({"eval", s1, "0"}), "works");
 
   const char* s2 = R"(
-  -- pragma: this-is-an-error
+  #!lua flags=this-is-an-error
```

Comment:
btw, does it mean it's an invalid lua code?

Reply:
Must be, it fails to compile. Also note that we insert it into a function when we store the script